ui_config/tenant/YJSG-Indore/routes/classroomForm.json

```
@@ -0,0 +1,92 @@
+{
+	"classroomFormConfig": {
+		"schema": {
+			"type": "object",
+			"properties": {
+				"classroomNo": {
+					"title": "Class Room No.",
+					"type": "string"
+				},
+				"level": {
+					"title": "Level",
+					"type": "string",
+					"enum": ["Level 1", "Level 2"]
+				},
+				"genderCategory": {
+					"title": "Gender Category",
+					"type": "string",
+					"enum": ["Male", "Female"]
+				},
+				"totalCapacity": {
+					"title": "Total Capacity",
+					"type": "string"
+				},
+				"averageAge": {
+					"title": "Average Age",
+					"type": "string"
+				},
+				"primaryTeacherAllocation": {
```

Remove allocation word from this and below entries
```suggestion
				"primaryTeacher": {
```